### PR TITLE
feat: Enhance news load resources using load-group - EXO-63805

### PR DESCRIPTION
**Pull request by hakermi**
Prior to this change, all news extensions were loaded inside all-bottom-container including the non needed once in all pages. This PR add the use of load-group mechanism to enhance the resources load and load only what is needed in the current page.

### CODE REVIEW

**Review by rdenarie — CHANGES_REQUESTED** [review]
Can you check the sonar feedback marked as BUG ?

File: services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java [hunk 1]
```diff
@@ -580,6 +580,13 @@ public Response getNewsByActivityId(@Parameter(description = "Activity id", requ
         return Response.status(Response.Status.NOT_FOUND).build();
       }
       news.setIllustration(null);
+      Identity userIdentity = identityManager.getOrCreateUserIdentity(currentIdentity.getUserId());
+      if (userIdentity != null) {
+        news.setFavorite(favoriteService.isFavorite(new Favorite("news",
+                                                                 news.getId(),
+                                                                 "",
+                                                                 Long.parseLong(userIdentity.getId()))));
+      }
```
Comment (rdenarie) [inline services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java:589]: This should be removed ?

File: webapp/src/main/webapp/WEB-INF/conf/news/nodetypes-templates/news/views/view1.gtmpl [hunk 2]
```diff
@@ -152,6 +152,7 @@
         activityId: '${activityId}',
         hiddenSpace: ${news.hiddenSpace},
         isSpaceMember: ${news.isSpaceMember},
+        favorite: ${news.isFavorite}
```
Comment (rdenarie) [inline webapp/src/main/webapp/WEB-INF/conf/news/nodetypes-templates/news/views/view1.gtmpl:155]: Why do you need this variable ?

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

**Comment by hakermi** [comment]
> Can you check the sonar feedback marked as BUG ?

Done

File: services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java (hunk 1 above)
Comment (hakermi) [reply services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java:589]: why it should be removed? i have faced an existing old issue and i fixed it.

File: webapp/src/main/webapp/WEB-INF/conf/news/nodetypes-templates/news/views/view1.gtmpl (hunk 2 above)
Comment (hakermi) [reply webapp/src/main/webapp/WEB-INF/conf/news/nodetypes-templates/news/views/view1.gtmpl:155]: https://github.com/exoplatform/news/pull/910#discussion_r1297132987

File: services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java (hunk 1 above)
Comment (hakermi) [reply services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java:589]: the issue is that when access news details and that you have set the news as favorite, you will still have the option to set it again and the icon of favorite is empty icon which indicates that news is not marked as favorite while it's the case

**Review by rdenarie — APPROVED** [review]
ok for me

**Pull request merged**